content/blog/2020/01/2020-01-27-jenkins-world-lisbon-with-love-from-india.adoc

```diff
@@ -0,0 +1,111 @@
+and help us with arranging everything for our travel. Thanks to him. :)
+
+== From India to Lisbon
+
+Abhyudaya and I boarded our flight from Indira Gandhi Airport (New Delhi) to Lisbon on December 1, 2019 morning at 0500 hours (local time). It was a fine
+trip with an hour layover in Istanbul Ataturk Airport. We arrived in Lisbon at 1500 hours (local time). The weather in Lisbon was terrific. A mild cold
+but strong sea breeze was the starting point of me falling in love with the place. We arrived at our hotel (Novotel Lisboa) in an Uber. Oleg met us at 
+the lobby to help us with check-in. It was great to finally meet him in person after months of knowing and working together. We had a good chat about the 
+event, what to expect and other sightseeing areas. After a short time of refresh, Long who travelled from Berlin a day before met us at the restaurant. We 
+had brief chat knowing each other, had our food and went to bed.
+
+== Day 0
+
+The next day (Dec 2) was Hackfest. We hit the bed after that as we had to reach Centro 
+de Congressos de Lisboa (CCL) where the event was organised by 0900 hours. I woke up early for a short jog in the streets. Lisbon is a city made on hills. 
+The streets have beautiful mosaic styled pavements. It was nice to see around the city. Then Abhyudaya and me went for breakfast and reached CCL in an 
+Uber at 0815 hours.
+
+image::/images/post-images/2020/01-gsoc-jenkins-world-parichay/with-oleg.JPG[A picture with Oleg Nenashev]
+
+image::/images/post-images/2020/01-gsoc-jenkins-world-parichay/sponsors.JPG[Sponsors]
+
+There was a round table sitting arrangement in an auditorium. It was like a meet-and-greet event to interact with other developers (some known and some new).
+Everybody had to figure out their problem statements and work on it. There was milk, juice, sandwiches which gave us energy throughout the day. I took a 
+small break to come out of the building to go to the other side of the road which was on the banks of Tagus River. From there you could have very close 
+view underneath the Ponte 25 de Abril (looks strikingly similar to Golden Gate Bridge). You can also see The Sanctuary of Christ the King on the other side 
+of the river (again looks similar to Christ the Redeemer in Rio, Brazil). It was great to kick off the event with Hackfest. At the end of Hackfest some of 
+us presentated their work. Later, we went to a nearby restaurant to have burgers which apparently was the best burger I ever had (could be because I hadn’t 
+tried too many burgers before :P). We talked and interacted with people from other parts of the world for about an hour and half then went back to our hotel 
+rooms.
+
+image::/images/post-images/2020/01-gsoc-jenkins-world-parichay/ponte-de-abril.jpg[Outside CCL]
+
+== Day 1
+
+The conference was scheduled next day. We woke up for breakfast and took the shuttle to CCL. We collected our t-shirts and IDs. The event managing team
```

Comment:
`The conference was scheduled next day` should be `The conference was scheduled next the day`